docs/pywbemclicmdshelp.rst

```diff
@@ -858,13 +861,12 @@ The following defines the help output for the `pywbemcli connection list --help`
 
       List the WBEM connection definitions.
 
-      List the WBEM connection definitions in the connections file as a table.
-
-      In the table, the current connection is marked with an "*" after its name.
-
-      Examples:
+      This command displays all entries in the connections file and the current
+      connection if it is not in the connections file as a table using the
+      command line output_format to define the table format.
```

Comment:
A current connection is not always defined, so the sentence should allow for that case as well.